src/main/java/ladder/domain/position/Position.java

```diff
@@ -0,0 +1,62 @@
+package ladder.domain.position;
+
+import ladder.domain.ladder.LadderSize;
+import ladder.exception.InvalidPositionException;
+
+import java.util.List;
+import java.util.Objects;
+
+public final class Position {
+    private static final int MIN_X = 0;
+
+    private final int x;
+    private final LadderSize ladderSize;
+
+    public Position(final int x, final LadderSize ladderSize) {
+        validate(x, ladderSize);
+        this.x = x;
+        this.ladderSize = ladderSize;
+    }
+
+    private void validate(final int x, final LadderSize ladderSize) {
+        if (x < MIN_X || x > ladderSize.getMaxWidthPosition()) {
+            throw new InvalidPositionException(x, MIN_X, ladderSize.getMaxWidthPosition());
+        }
+    }
+
+    public Position move(List<Boolean> lines) {
+        int leftPos = x - 1;
+        int rightPos = x + 1;
+        int leftLine = x - 1;
+        int rightLine = x;
+
+        if (leftPos >= MIN_X && lines.get(leftLine)) {
+            return moveLeft(x);
+        } else if (rightPos <= ladderSize.getMaxWidthPosition() && lines.get(rightLine)) {
```

Comment:
```suggestion
        }
        if (rightPos <= ladderSize.getMaxWidthPosition() && lines.get(rightLine)) {
```
객체지향 생활체조 원칙에 따라 else는 제거해주세요!